crates/fj-proc/src/lib.rs

```diff
@@ -148,6 +148,18 @@ pub fn model(_: TokenStream, input: TokenStream) -> TokenStream {
                 ),
             );
 
+            let attrs = item.attrs;
+            let vis = item.vis;
+            let sig = item.sig;
+            let statements = item.block.stmts;
+
+            let item = quote::quote! {
+                #(#attrs)* #vis #sig {
+                    fj::abi::initialize_panic_handling();
+                    #(#statements)*
+                }
+            };
+
```

Comment:
You seem to be adding the new and improved code here, without removing the old and busted one. `initialize_panic_handling` says it can be called multiple times, but I think that's not enough reason to keep the old code around :joy:

Reply:
I Had removed the old version and that println but for some reason merging brought a lot of the old stuff back. I missed those two.
I'm going to have to give this re-basing thing a try.

I'll try to fix that up today.